docs/outlook/spam-reporting.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Implement an integrated spam reporting add-in (preview)
```

Comment:
Consider hyphenating "spam reporting" when it's used as an adjective - see the [Hyphen](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765) article in the style guide
```suggestion
title: Implement an integrated spam-reporting add-in (preview)
```